tasks/computer-vision/image-classification/MLproject

```diff
@@ -0,0 +1,12 @@
+name: pneumonia_model
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      batch_size: {type: int, default: 32}
+      img_size: {type: int, default: 64}
+	  epoch: {type: int, default: 20}

```

Comment:
@bilalmoiz please convert your tabs to spaces (lines 10 and 11) for MacOS compatibility

Reply:
Hello, 

I took care of it. Please let me know if there's anything else you need.

Thanks,
Bilal